### PR TITLE
Changed default value of provider to 'Meshery'

### LINKED ISSUE #7509: [mesheryctl] system provider enhancement
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior

* Responses given by `mesheryctl system provider` when the meshery server is not running are vague
* Default meshconfig has no `provider` value set

#### Desired Behavior

- [x] Improve feedback messages when Meshery server is not running 
- [x] Set default provider value in meshconfig to `Meshery`

#### Mockups

---

#### Contributor Resources

- ⌨️ [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by abdullah1308**
**Notes for Reviewers**

This PR fixes #7509 

1. Value of `provider` in meshconfig is `Meshery` by default
2. Changed feedback messages for `system provider list` and `system provider set` when Meshery server is unreachable

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/system/provider.go [hunk 1]
```diff
@@ -305,7 +305,7 @@ mesheryctl system provider switch [provider]
 var resetProviderCmd = &cobra.Command{
 	Use:   "reset",
 	Short: "reset provider to default",
-	Long:  "Reset provider of context in focus to default (No provider set)",
+	Long:  "Reset provider of context in focus to default (Meshery)",
```
Comment (MUzairS15) [inline mesheryctl/internal/cli/root/system/provider.go:308]: Above language seems confusing.

“Reset provider for current context to default (Meshery)”.

Comment (abdullah1308) [reply mesheryctl/internal/cli/root/system/provider.go]: Changed

Comment (MUzairS15) [reply mesheryctl/internal/cli/root/system/provider.go]: @abdullah1308 I meant `context in focus` ----> `current context`
`Long:  "Reset provider of current context to default (Meshery)",`

Comment (abdullah1308) [reply mesheryctl/internal/cli/root/system/provider.go]: Done

**Review by MUzairS15 — APPROVED** [review]

**Comment by Chadha93** [comment]
@abdullah1308  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Review by leecalcote — APPROVED** [review]

**Pull request merged**